package.json

```diff
@@ -36,23 +37,24 @@
     "mustache": "^3.2.1",
     "particle-api-js": "^7.4.0",
     "pg": "^7.18.2",
+    "react": "^17.0.2",
```

Comment:
Why did you need to add this?

Reply:
I must have added this while debugging the node.js versioning issue following a stack overflow post. I'll remove it